### DISCUSSION TITLE
CLI: CSV Loading via stdin

### DISCUSSION THREAD

**Discussion by aborruso** [duckdb/duckdb#4619] [Q&A] [answered]
Hi,
is there a way to load a csv via stdin?

Something like

```
myscript.sh | duckdb --csv  -c "SUMMARIZE SELECT * FROM -"
```

The output of `myscript.sh ` is a CSV.

Thank you

**Comment by Alex-Monahan** [comment, marked as answer]
Would this work for you?
#2844

**Reply by aborruso** [reply]
thank you

**Reply by aborruso** [reply]
I have opened a PR about it https://github.com/duckdb/duckdb-web/pull/292

**Reply by aborruso** [reply]
Using SUMMARIZE, I have some problem https://github.com/duckdb/duckdb/pull/2844#issuecomment-1237715548
